01-structural-imbalance-overview.ipynb

```diff
@@ -379,22 +397,27 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For a submission on February 13, 2020, the minor-embedding for the entire saved database (238 variables) required about half a D-Wave 2000Q QPU's qubits. Ocean software's [problem inspector](https://docs.ocean.dwavesys.com/en/stable/docs_inspector/sdk_index.html) tool shows the embedded problem: \n",
+    "For one particular submission to an Advantage quantum computer, the minor-embedding for the entire saved database (238 variables) represented one variable with a dozen qubits coupled strongly together (a [chain](https://docs.ocean.dwavesys.com/en/stable/concepts/embedding.html) of qubits). Chains are used when minor-embedding any but the smallest problems; however, such long chains can be unreliable and affect performance. \n",
+    "\n",
+    "Ocean software's [problem inspector](https://docs.ocean.dwavesys.com/en/stable/docs_inspector/sdk_index.html) tool shows the embedded problem, highlighting the long chain: \n",
     "\n",
     "<img src=\"images/EmbeddingFullDB_inspector.png\" width=700x/>"
```

Comment:
Did you mean to use `px` for length unit? I don think `x` is a valid [CSS unit](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#numbers_lengths_and_percentages).
```suggestion
    "<img src=\"images/EmbeddingFullDB_inspector.png\" width=\"700px\"/>"
```

Reply:
It looks like JNs accept `width=<number><anything goes>` here but the [docs](https://jupyterbook.org/en/stable/content/figures.html) do say `px`.